llama_toolchain/agentic_system/meta_reference/system_prompt.py

```diff
@@ -34,28 +40,54 @@ def get_agentic_prefix_messages(
             ]
         )
         if tool_str:
-            content += f"Tools: {tool_str}\n"
+            content += f"Tools: {tool_str}"
 
     current_date = datetime.now()
     formatted_date = current_date.strftime("%d %B %Y")
     date_str = f"""
 Cutting Knowledge Date: December 2023
-Today Date: {formatted_date}\n\n"""
+Today Date: {formatted_date}\n"""
     content += date_str
 
     if custom_tools:
-        custom_message = get_system_prompt_for_custom_tools(custom_tools)
-        content += custom_message
-
-    # TODO: Replace this hard coded message with instructions coming in the request
-    if False:
-        content += "You are a helpful Assistant."
+        if tool_prompt_format == ToolPromptFormat.function_tag:
```

Comment:
oi, so `function_tag` results in this going into the system message and `json` in a user message? very intentional?

Reply:
I am making this also a user message for consistency since i tested and looks like both operate similarly.